### PR TITLE
feat: Review submitted app version and validate info

### PR DESCRIPTION
**Pull request by whDongRui**
solve #523 -- Version submit review and info check

### CODE REVIEW

File: src/pages/Dashboard/Apps/Info/index.jsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import {
 import Layout, { Card } from 'components/Layout';
 import DetailTabs from 'components/DetailTabs';
 
+import PropTypes from 'prop-types';
```
Comment (iwisunny) [inline src/pages/Dashboard/Apps/Info/index.jsx]: third party lib should lift up, place it after react

File: src/stores/app/index.js [hunk 2]
```diff
@@ -257,26 +286,40 @@ export default class AppStore extends Store {
...
-        category_id: this.appDetail.category_id
-      })
+      _.assign(
+        data,
+        {
+          app_id: this.appDetail.app_id,
+          category_id: this.appDetail.category_id,
+          icon: this.appDetail.icon
+        },
+        Boolean(e)
```
Comment (iwisunny) [inline src/stores/app/index.js]: what does this `Boolean(e)`?  e is event ?

File: src/stores/app/index.js [hunk 3]
```diff
@@ -257,26 +286,40 @@ export default class AppStore extends Store {
...
-    if (get(this.createResult, 'app_id')) {
+    if (hasNote && get(this.createResult, 'app_id')) {
       this.info('应用信息保存成功');
     }
   };
 
   @action
   modifyApp = async e => {
-    e.preventDefault();
+    if (e) {
```
Comment (iwisunny) [inline src/stores/app/index.js]: what if e is not event object ?

File: src/pages/Dashboard/Apps/Info/index.jsx (hunk 1 above)
Comment (whDongRui) [reply src/pages/Dashboard/Apps/Info/index.jsx]: ok

File: src/stores/app/index.js (hunk 3 above)
Comment (whDongRui) [reply src/stores/app/index.js]: The app version submit will used this Info page, the LayoutStepper Component next step will used this modifyApp.

File: src/stores/app/index.js (hunk 2 above)
Comment (whDongRui) [reply src/stores/app/index.js]: e is event ,transfer this  param for the Stepper Component not used notification.

**Pull request merged**